Bitcoin/0.18.0/docker-entrypoint.sh

```diff
@@ -15,6 +15,14 @@ if [[ "$1" == "bitcoin-cli" || "$1" == "bitcoin-tx" || "$1" == "bitcoind" || "$1
 		CONFIG_PREFIX=$'mainnet=1\n[main]'
 	fi
 
+	if [[ $BITCOIN_WALLETDIR ]]; then
```

Comment:
You should do the same things that you did with the other docker repository (elif/else/check that BITCOIN_NETWORK is specified)